sui_programmability/examples/games/sources/vdf_based_lottery.move

```diff
@@ -0,0 +1,160 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
Please move this example following https://github.com/MystenLabs/sui/tree/c2f099f15f005e8ceeeee8386ae23f9c928d4c8e/sui_programmability/examples